data_usage.go

```diff
@@ -143,34 +155,43 @@ func (s *DataUsageSyncer) SyncDataUsage(config *data_usage.DataUsageSyncConfig)
 			if returnedRow.SchemaName.Valid {
 				schemaName = returnedRow.SchemaName.String
 			}
-			accessedDataObjects := common.ExtractInfoFromQuery(returnedRow.Query, databaseName, schemaName)
 
-			if len(accessedDataObjects) == 0 {
-				logger.Debug(fmt.Sprintf("No data objects returned for query: %s, batch %d", returnedRow.Query, currentBatch))
+			accessedDataObjects, localErr := common.ParseSnowflakeInformation(returnedRow.Query, databaseName, schemaName,
+				returnedRow.BaseObjectsAccessed, returnedRow.DirectObjectsAccessed, returnedRow.ObjectsModified)
+
+			if localErr != nil {
+				// TODO: add logic to include query
```

Comment:
Is there a ticket for this?

Reply:
Yes, there's a pre-existing ticket; https://github.com/raito-io/global/issues/69. 

The reason not to do it 'quickly' in this PR as well is because it needs to be coordinated from different repos (the struct that contains the data written to the import file is in the plugin repo).